evalml/pipelines/components/transformers/transformer.py

```diff
@@ -34,17 +35,17 @@ def transform(self, X, y=None):
         Returns:
             ww.DataTable: Transformed X
         """
+        X_ww = _convert_to_woodwork_structure(X)
```

Comment:
I decided to move these out of the try/except because I think it's more clear what the try/except is for. If it errors out while doing Woodwork conversions (as it did while I was testing stuff), it shouldn't throw a `MethodPropertyNotFoundError` :)